app/forms/nominate_induction_tutor_form.rb

```diff
@@ -44,13 +37,38 @@ def existing_name
     existing_user.full_name
   end
 
+  def name_different?
+    existing_name.present? && existing_name != full_name
+  end
+
+  def email_already_taken?
+    ParticipantProfile.active_record.ects.joins(:user).where(user: { email: }).any?
```

Comment:
Not sure if we should go further here because a SIT won't have a participant profile unless they are also a Mentor?

Reply:
@tonyheadford if I change this line with `Identity.find_user_by(email:).present?` then most of the tests in `spec/requests/admin/schools/induction_coordinators_spec.rb` fail.
I'm not sure it's safe to change. Any idea?

Reply:
Whenever we check for the existence of an email we have to check the participant identities as well as the User.email, all of which `Identity.find_user_by(email:)` does correctly.  I'm not sure which specs are failing but I would suspect that there are errors somewhere in those spec's setup.